.github/workflows/main.yml

```diff
@@ -153,7 +156,7 @@ jobs:
           --print-address
       - name: Run stratis-cli tests
         run: >
-          RUST_LOG=stratisd=debug
+          RUST_LOG=trace
```

Comment:
For stratis-cli tests, might as well stick with DEBUG level. They are run against the sim engine, and so don't tend to cause udev events of interest.

Reply:
Thanks - I will make that change on my branch, and disable the other tests that I'm not concerned with to keep the runs to a minimum.